Desktop/gui/preferencesmodel.h

```diff
@@ -243,6 +244,7 @@ private slots:
 					_allResultFonts,
 					_allCodeFonts;
 	bool			_githubPatCustom; //Should be initialized on prefs construction
+	QString			_developerFolder; //If empty, this is the Developer folder Setting.
```

Comment:
and of course why?
Explain that it is used for generating wrapper through jasp started with some arguments because this comment as it stands explains very little...